app/src/main/res/layout/item_right_convention.xml

```diff
@@ -43,12 +52,11 @@
             android:id="@+id/edit_conversation"
             android:layout_width="0dp"
             android:layout_height="wrap_content"
-            android:layout_marginBottom="@dimen/dp_8"
             android:layout_marginEnd="@dimen/dp_8"
-            android:layout_marginStart="@dimen/dp_8"
-            android:layout_marginTop="@dimen/dp_8"
+            android:layout_marginStart="@dimen/dp_4"
```

Comment:
ERROR: [RtlCompat] To support older versions than API 17 (project specifies 16) you should *also* add `android:layout_marginLeft="@dimen/dp_4"`
 API 17 adds a `textAlignment` attribute to specify text alignment. However, if you are supporting older versions than API 17, you must *also* specify a gravity or layout_gravity attribute, since older platforms will ignore the `textAlignment` attribute.